tests/event_consumer_integration.rs

```diff
@@ -303,7 +303,8 @@ async fn test_nack_and_rereceive() -> Result<()> {
         "--ctl-port",
         &ctl_port,
     ])
-    .await;
+    .await
+    .unwrap();
```

Comment:
nit: this could be an `expect` like the other wash helpers

Reply:
Good call, thanks!